test/test_script.py

```diff
@@ -47,13 +49,17 @@
     ScriptExpression,
     ScriptFunction,
     ScriptParser,
+    ScriptRuntimeError,
     ScriptSyntaxError,
     ScriptUnknownFunction,
     register_script_function,
     script_function,
 )
 
 
+IS_WIN = os.name == 'nt'
```

Comment:
Is there a specific reason to not use `IS_WIN` from `picard.const.sys`?

Reply:
No reason other than I forgot there was one.  Fixed now.  Thanks.